pkg/restapi/operation/didexchange/didexchange_test.go

```diff
@@ -319,9 +318,28 @@ func TestServiceEvents(t *testing.T) {
 	didExSvc, err := didexsvc.New(&did.MockDIDCreator{}, &protocol.MockProvider{CustomStore: store})
 	require.NoError(t, err)
 
+	done := make(chan struct{})
+
 	// create the client
 	op, err := New(&mockprovider.Provider{StorageProviderValue: mockstore.NewMockStoreProvider(),
-		ServiceValue: didExSvc}, webhook.NewHTTPNotifier(nil))
+		ServiceValue: didExSvc},
+		&mockNotifier{
+			notifyFunc: func(topic string, message []byte) error {
+				require.Equal(t, "connections", topic)
```

Comment:
`"connections"` should probably be a constant

Reply:
https://github.com/hyperledger/aries-framework-go/pull/580#discussion_r338765349